src/Microsoft.AspNetCore.Mvc.RazorPages/PageModel.cs

```diff
@@ -1614,6 +1615,27 @@ public virtual ObjectResult StatusCode(int statusCode, object value)
         public virtual UnauthorizedResult Unauthorized()
             => new UnauthorizedResult();
 
+        /// <summary>
+        /// Creates a <see cref="PartialViewResult"/> by specifying the name of a partial to render.
+        /// </summary>
+        /// <param name="partialViewName">The partial name.</param>
+        /// <returns>The created <see cref="PartialViewResult"/> object for the response.</returns>
+        public virtual PartialViewResult Partial(string partialViewName)
```

Comment:
This should be called `PartialView` to keep inline with Controller: https://github.com/aspnet/Mvc/blob/master/src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs#L174. Also `partialViewName` -> `viewName` to keep the signatures similar

Reply:
Sure